api.yaml

```diff
@@ -140,6 +209,39 @@ components:
           enum: [ADDED, DELETED]
       required:
       - changeType
+    CloudAssetDetails:
+      type: object
+      properties:
+        privateIpAddresses:
+          type: array
+          items:
+            type: string
+        publicIpAddresses:
+          type: array
+          items:
+            type: string
+        hostnames:
+          type: array
+          items:
+            type: string
+        createdAt:
+          type: string
+          format: date-time
+        deletedAt:
```

Comment:
What are createdAt and deletedAt intended to convey here? Are they proxies for when a system came online or went offline?

Reply:
These have been removed.